ocs_ci/helpers/helpers.py

```diff
@@ -2995,6 +2995,25 @@ def get_mon_pdb():
     return disruptions_allowed, min_available_mon, max_unavailable_mon
 
 
+def verify_pdb_mon(disruptions_allowed, max_unavailable_mon):
+    """
+    Verify PDB mon
+
+    Args:
+        disruptions_allowed (int): the expected number of disruptions_allowed
+        max_unavailable_mon (int): the expected number of max_unavailable_mon
+
+    """
+    logging.info("Check mon pdb status")
+    mon_pdb = get_mon_pdb()
+    assert (
+        disruptions_allowed == mon_pdb[0]
+    ), f"disruptions_allowed expected is {disruptions_allowed} actual is {mon_pdb[0]}"
+    assert (
```

Comment:
throw exception here replacing assert

Reply:
I think that the assertion exception is good too, just needs a bit of a rephrase:
```
f"The expected allowed disruption is: {disruptions_allowed}. The actual one is {mon_pdb[0]}"
```

Reply:
done

Reply:
Do you think it is better we return bool here instead of throwing assert in the function? the function handling can be done in the test case based on the return value

Reply:
done